### PR TITLE
State machine

### PR DESCRIPTION
**Pull request by moklidia**

### CODE REVIEW

File: test/models/admin_test.rb [hunk 1]
```diff
@@ -0,0 +1,7 @@
+require 'test_helper'
+
+class AdminTest < ActiveSupport::TestCase
+  # test "the truth" do
```
Comment (artempartos) [inline test/models/admin_test.rb]: the same about model tests

File: app/models/task.rb [hunk 2]
```diff
@@ -6,4 +6,33 @@ class Task < ApplicationRecord
 	validates :author, presence: true
 	validates :description, length: { maximum: 500 }
 
+	state_machine :initial => :new_task do
+		event :start do
```
Comment (artempartos) [inline app/models/task.rb]: something wrong with indentations

Comment (moklidia) [reply app/models/task.rb]: Fixed!

File: test/models/admin_test.rb (hunk 1 above)
Comment (moklidia) [reply test/models/admin_test.rb]: Done!

File: test/factories/users.rb [hunk 3]
```diff
@@ -1,10 +1,12 @@
...
+
 FactoryBot.define do
   factory :user do
-    first_name { "MyString" }
-    last_name { "MyString" }
-    password { "" }
-    email { "MyString" }
-    avatar { "MyString" }
-    type { "" }
+    first_name { 'MyString' }
```
Comment (artempartos) [inline test/factories/users.rb]: the same about factories

Comment (moklidia) [reply test/factories/users.rb]: Updated in the "models" branch

**Comment by artempartos** [comment]
👍

**Pull request merged**